seed/src/event.rs

```diff
@@ -36,6 +41,15 @@ pub enum Event {
     PeerDisconnected(PeerId),
     /// A project has been tracked from a peer.
     ProjectTracked(Project, PeerId),
+    /// A project has been updated from a peer.
+    ProjectUpdated {
+        /// The URN of the updated project, including the updated path.
+        urn: RadUrn,
+        /// The updated rev, if any.
+        rev: Option<Rev>,
```

Comment:
Not sure in what cases this is `None` :thinking:

Reply:
If it's `None` then it's basically just advertising the project. I'm not even sure what the protocol would do in that case. It'll possibly just mark it as uninteresting